src/OpenLoco/src/Paint/PaintTile.cpp

```diff
@@ -66,18 +66,17 @@ namespace OpenLoco::Paint
     // 0x00461EA7
     static void paintConstructionArrow(PaintSession& session, const World::Pos2& loc)
     {
-        static loco_global<World::Pos3, 0x00F24942> _constructionArrowLocation;
-        static loco_global<uint8_t, 0x00F24948> _constructionArrowDirection;
+        auto& arrow = getConstructionArrow();
         if (!World::hasMapSelectionFlag(World::MapSelectionFlags::enableConstructionArrow))
         {
             return;
         }
-        if (session.getUnkPosition() != _constructionArrowLocation)
+        if (session.getUnkPosition() != arrow.pos)
```

Comment:
Can't check right now, but is this UnkPosition always related to the construction arrow?